File: modul6/C/PRAK601-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>

int main(){
    int i, j, baris, kolom;
    printf("Jumlah baris dan kolom :\n");
    scanf("%d %d", &baris, &kolom);
    int limitmatriks[baris][kolom];
    printf("Input matriks :\n");
    for(i = 0; i < baris; i++){
        for(j = 0; j < kolom; j++){
            scanf("%d", &limitmatriks[i][j]);
        }
    }
    printf("Output matriks :\n");
    for(i = 0; i < baris; i++){
        for(j = 0; j < kolom; j++){
            printf("%d ", limitmatriks[i][j]);
        }
        printf("\n");
    }
return 0;
}

File: modul6/Python/PRAK601-2310817210023-RifkyPutraMahardika.py
baris, kolom = map(int, input("Jumlah baris dan kolom :\n").split())
print("Input matriks :")
matriks = list(map(int, input().split()))

print("Output matriks :")
for i in range(baris):
    for j in range(kolom):
        print(matriks[i * kolom + j], end=" ")
    print()